corehq/apps/data_dictionary/urls.py

```diff
@@ -11,7 +12,8 @@
     url(r"^generate/$", generate_data_dictionary),
     url(r"^json/$", data_dictionary_json, name='data_dictionary_json'),
     url(r"^json/?(?P<case_type_name>\w+)/?$", data_dictionary_json, name='case_type_dictionary_json'),
-    url(r"^update_case_property/$", update_case_property, name='update_case_property'),
+    url(r"^update_case_properties/$", update_case_property, name='update_case_property'),
```

Comment:
The naming here is fairly confusing: there's an `update_case_property` url and an `update_case_property` view but they are two different things.